doc/cabal-project.rst

```diff
@@ -32,6 +32,8 @@ options):
 
 4. ``cabal.project.local`` (the output of ``cabal v2-configure``)
 
+A call to ``cabal build`` will consider ``cabal.project*`` from parent 
```

Comment:
Just some wording suggestion:
_Any_ call to ... will consider ... _files_ from parent ...

Reply:
Done, thanks!